@ornikar/jest-config-react/__mocks__/@storybook/react.js

```diff
@@ -71,10 +75,9 @@ exports.storiesOf = (groupName) => {
           await act(async () => {
             const rtlApi = render(story(context), { wrapper: wrappingComponent });
             const { unmount, asFragment } = rtlApi;
-            // https://www.apollographql.com/docs/react/development-testing/testing/#testing-final-state
-            // delays until the next "tick" of the event loop, and allows time
-            // for that Promise returned from MockedProvider to be fulfilled
-            await (waitForExpectation ? waitFor(() => waitForExpectation(rtlApi, expect)) : wait(0));
+            if (waitForExpectation) {
```

Comment:
this should be breaking ? even though it's a small change, you would still have to change the places where it's used

Reply:
https://github.com/ornikar/learner-apps/pull/3118/files#diff-ef9d67fe2945e43b4d5a03adbab374226ea5ceac7bc793c6dcc5b921dd7b06a7R209-R214

Reply:
yes it should